### PR TITLE
Add itest for multi unit alerts

### PR DESCRIPTION
**Pull request by sed-i**
This PR adds an itest that addresses https://github.com/canonical/prometheus-k8s-operator/pull/239:
Now that `juju_unit` is no longer added to rule files, this test confirms that alerts still fire per unit (not only from the leader unit), and are correctly labeled as such.

### CODE REVIEW

**Review by Abuelodelanada — APPROVED** [review]

**Review by mmanciop — CHANGES_REQUESTED** [review]
Red tests are red

**Comment by rbarry82** [comment]
> Red tests are red

No pebble socket. Possible juju failure. Rerunning

**Comment by sed-i** [comment]
> Red tests are red

This passed locally but now here fails for

```
  Full diff:
  - ['avalanche/0', 'avalanche/1']
  + []
```

i.e. seems like no `juju_unit` labels on the alerts, in contrast to expectation. Investigating.

**Comment by sed-i** [comment]
This test is flaky because the charm I uploaded to edge has a stray rule file:
https://github.com/canonical/charmcraft/issues/710

File: tox.ini [hunk 1]
```diff
@@ -67,8 +67,8 @@ commands =
 description = Run integration tests
 deps =
     jinja2
-    #git+https://github.com/juju/python-libjuju.git
-    juju
+    git+https://github.com/juju/python-libjuju.git
```
Comment (mmanciop) [inline tox.ini:70]: What’s with this, this time? :-)

**Review by mmanciop — APPROVED** [review]

File: tox.ini (hunk 1 above)
Comment (sed-i) [reply tox.ini:70]: `block_until_with_coroutine` is not released yet.

**Pull request merged**